models/release.go

```diff
@@ -244,6 +244,15 @@ func GetReleasesByRepoID(repoID int64, page, pageSize int) (rels []*Release, err
 	return rels, err
 }
 
+// GetReleaseCountByRepoID returns the count of releases of repository
+func GetReleaseCountByRepoID(repoID int64, isOwner bool) (total int64, err error) {
+	if isOwner {
+		return x.Where("repo_id = ?", repoID).Count(&Release{})
+	}
+
+	return x.Where("repo_id = ? AND is_prerelease = 0 AND is_draft = 0", repoID).Count(&Release{})
```

Comment:
Also this probably wont work in postgres as I think xorm creates bool columns in it.